patterns/rightsizing-virtual-machines.md

```diff
@@ -0,0 +1,51 @@
+# Rightsizing oversized VMs
+
+## Version
+1.0
+
+## Submitted By
+Bill Johnson (@dubrie)
+
+## Published Date
+TBD
+
+## Intent
+
+Choose servers that are the right size for the task at hand.
+
+## Tags
+---
+tags:
+ - cloud
+ - compute
+ - role-cloud-engineer
+ - size-small
+---
+
+## Problem
+
+It's better to have one VM running at a higher utilization than two running at low utilization rates. This is both from an energy proportionality angle. 2 servers running at low utilization rates will consume more energy than one server running at a high rate of utilization. But also, from an embodied carbon angle, the unused capacity on the underutilized server could be more efficiently used for another task or process.
+
+## Solution
+
+Change the number of resources allocated to a VM to match the utilization requirements of the VM. For example, adding a vCPU if the VM is running high CPU utilization or removing memory if the server is not using all of its allocated memory.
+
+Select a pre-configured server that better matches the utilization requirements of the process.
```

Comment:
Should we split this into a separate pattern?

Reply:
Do you feel there is enough difference between them? I thought about splitting them initially but ended up keeping it together since both parts are adjusting and optimizing the utilization of the machine, so it seemed duplicative to have this defined in two different patterns. I think all the other sections would be the same if it was split into two.

Reply:
Updated into 2 different patterns!